### PR TITLE
#722 Documentation for the Shift Deadline plugin

### PR DESCRIPTION
**Pull request by leegeertseninbibo**
This is the documantation on how to set up the Shift Deadline plugin.

Issue: #72

### CODE REVIEW

File: docs/integration_resources/software/deadline.md
Comment (juliavisserinbibo) [inline docs/integration_resources/software/deadline.md:1]: Can you make this impersonal? Not using the pronoun "you"

**Review by juliavisserinbibo — CHANGES_REQUESTED** [review]

File: docs/integration_resources/software/deadline.md
Comment (leegeertseninbibo) [reply docs/integration_resources/software/deadline.md:1]: Fone

File: docs/integration_resources/software/deadline.md [hunk 1]
```diff
@@ -1,5 +1,63 @@
 # Deadline
 
-Shift integration in Deadline is on the works!
+Shift integration in Deadline is in the works. The Shift plugin for Deadline is currently only available on Windows and supports execution of Shift workflows in batch mode or within Maya.
```
Comment (juliavisserinbibo) [inline docs/integration_resources/software/deadline.md:3]: Given that we are already documenting Deadline, is it necessary to keep "Shift integration in Deadline is in the works?

```suggestion
Shift integration in Deadline is in the works. The Shift plugin for Deadline is currently only available on Windows and supports the execution of Shift workflows in batch mode or within Maya.

```

File: docs/integration_resources/software/deadline.md [hunk 2]
```diff
@@ -1,5 +1,63 @@
...
\ No newline at end of file
+# Shift Plugin for Deadline Renderfarm
+
+This guide explains how to set up the Shift plugin for Deadline. The setup process varies depending on the version of Python used by the Deadline installation.
+
+## Prerequisites
+
+- Access to the Deadline repository.
+- Administrative rights to create directories and modify settings in the Deadline Monitor.
+- The Shift software installed.
```
Comment (juliavisserinbibo) [inline docs/integration_resources/software/deadline.md:13]:
```suggestion
- The Shift software installed and activated.

```

File: docs/integration_resources/software/deadline.md [hunk 3]
```diff
@@ -1,5 +1,63 @@
...
+
+1. Navigate to the Deadline repository at `<path_to_deadline_repository>/custom/plugins`.
+2. Create a new folder named `Shift`.
+
+## Step 2: Copy the Plugin Files
+
+1. Locate the `deadline` folder in the Shift installation directory.  
+   The path is `<path_to_the_shift_installation_folder>/deadline`.
+2. Inside the `deadline` folder, there are two directories:
+   - `python2.7`
```
Comment (juliavisserinbibo) [inline docs/integration_resources/software/deadline.md:25]: We should remove python 2.7 right?

**Review by juliavisserinbibo — CHANGES_REQUESTED** [review]

File: docs/integration_resources/software/deadline.md [hunk 4]
```diff
@@ -1,5 +1,63 @@
...
+To set environment variables for Shift while running on Deadline, two options are available:
+
+### Option 1: Modify `PluginPreLoad.py`
+
+1. Open the `PluginPreLoad.py` script from `<path_to_deadline_repository>/custom/plugins/Shift`.
+2. Add or modify any necessary environment variables.
+
+### Option 2: Use a Custom PreLoad Script
+
+A custom Python file can be created to set up environment variables, utilizing the following environment variable:
```
Comment (hectorjimenezinbibo) [inline docs/integration_resources/software/deadline.md:54]:
```suggestion
A custom Python file can be created to set up environment variables, using the following environment variables:
```

**Review by hectorjimenezinbibo — CHANGES_REQUESTED** [review]

File: docs/integration_resources/software/deadline.md (hunk 1 above)
Comment (leegeertseninbibo) [reply docs/integration_resources/software/deadline.md]: Yes, we should mention that it is in the works, right now we have the plugin so we can send a Shift workflow to Deadline. But we have no way to send any other jobs to deadline from Shift for the moment

File: docs/integration_resources/software/deadline.md (hunk 3 above)
Comment (leegeertseninbibo) [reply docs/integration_resources/software/deadline.md:27]: No, Shift uses python 3. But some companies might still have their Deadline working with python 2

**Review by juliavisserinbibo — APPROVED** [review]

**Review by hectorjimenezinbibo — APPROVED** [review]

**Review by albertochimentiinbibo — APPROVED** [review]

**Pull request merged**